feature/KAS-3190 Pub v3: /proofs: publication request from proof receipt

mdv-professional opened the pull request:
[kasticketje](https://kanselarij.atlassian.net/browse/KAS-3190)

---

#### Out of scope
##### Gedaan
- [x] (onvolkomenheid UI) request modal sluit nu meteen na save
- [x] Ik heb begrepen waarom ik onder `publicaties/:id/dossier` en `./besluiten` `@model` moest gebruiken i.p.v. this.model: setupController vereist een call super.setupController(...)
##### Niet gedaan
eigenlijk binnen scope, maar ik wilde niet afwijken van wat in de ProofRequestModal is gedaan:
- [x] Ik had de naamgeving iets eenduidiger willen maken:
    - `uploadedPieces` in parameter van `onSave` => hernoemen naar `pieces`
           naamgeving dekt de lading niet meer
    - de `save` roept een methode `onSave`, `onCreatePublicationRequest`, `createPublicationRequest` en `createPublicationRequestActivity` => alles hernoemen naar `(on)CreatePublicationRequest`

Er zijn me nog een aantal gebrekkigheden opgevallen buiten scope. Ik ga ze toevoegen aan mijn overzichtje: [Publicatie-flows v3: onvolkomenheden.md](https://gist.github.com/mdv-professional/02982d23b3a35e51852e819a1e4e9d08#file-publicatie-flows-v3-onvolkomenheden-md)

Commit:
show from ProofReceivedPanel PublicationRequestModal
KAS-3190 pub: v3: proofs: request publication from ProofReceivedPanel

Commit:
(lint-fix) ProofReceivedPanel
KAS-3190 pub: v3: proofs: request publication from ProofReceivedPanel

Commit:
show transferred pieces / PublicationRequestModal
KAS-3190 pub: v3: proofs: request publication from ProofReceivedPanel

Commit:
unlink transferred pieces / PublicationRequestModal
KAS-3190 pub: v3: proofs: request publication from ProofReceivedPanel

Commit:
(lint-fix) publication-request-modal.js
KAS-3190 pub: v3: proofs: request publication from ProofReceivedPanel

Commit:
(lint-fix)
KAS-3190 pub: v3: proofs: request publication from ProofReceivedPanel

Commit:
save publication request / ./proofs
KAS-3190 pub: v3: proofs: request publication from ProofReceivedPanel

Commit:
(comment)
KAS-3190 pub: v3: proofs: request publication from ProofReceivedPanel

Commit:
(lint-fix) publication-service.js
KAS-3190 pub: v3: proofs: request publication from ProofReceivedPanel

Commit:
(lint-fix) PublicationProofsRoute
KAS-3190 pub: v3: proofs: request publication from ProofReceivedPanel

Commit:
disable publication request when already requested | /publications/:id/proofs
KAS-3190 pub: v3: proofs: request publication from ProofReceivedPanel

Commit:
close Proof- and PublicationRequestModal after save | /publications/:id/translations|proofs
KAS-3190 pub: v3: proofs: request publication from ProofReceivedPanel

Commit:
only attach proof result pieces | ./proofs
KAS-3190 pub: v3: proofs: request publication from ProofReceivedPanel

Commit:
Merge branch 'development' into feature/KAS-3190-pub-v3--proofs-pub-req

Commit:
prevent delete of pieces linked to a proof | ./publication
KAS-3190 pub: v3: proofs: request publication from ProofReceivedPanel

Commit:
don't fetch used-pieces | ./publication
reason: not used
KAS-3190 pub: v3: proofs: request publication from ProofReceivedPanel

Commit:
show loader while loading in PublicationRequestModal | ./publication
KAS-3190 pub: v3: proofs: request publication from ProofReceivedPanel

Commit:
Merge branch 'development' into feature/KAS-3190-pub-v3--proofs-pub-req

Commit:
(translation) avoid Publicatie voor publicatie, without own term
reason: "Publicatie voor publicatie" sounds confusing and "Publicatieproces" I made up myself
KAS-3190 pub: v3: proofs: request publication from ProofReceivedPanel

Commit:
(comment) remove confusing comment
reason: no unlinking is necessary, since related models are deleted
KAS-3190 pub: v3: proofs: request publication from ProofReceivedPanel

Commit:
(lint-fix)
KAS-3190 pub: v3: proofs: request publication from ProofReceivedPanel

Commit:
fix: typo "proofingActivty" in PublicationRequestModal
KAS-3190 pub: v3: proofs: request publication from ProofReceivedPanel

Commit:
(refactor) add method Store.count(...)
KAS-3190 pub: v3: proofs: request publication from ProofReceivedPanel

Commit:
(aside/bugfix) use super.setupController and this.model
KAS-3190 pub: v3: proofs: request publication from ProofReceivedPanel

Commit:
Merge branch 'development' into feature/KAS-3190-pub-v3--proofs-pub-req

Commit:
add sorting + comment | PublicationRequestModal
KAS-3190 pub: v3: proofs: request publication from ProofReceivedPanel

erikap committed:
Align proof-request-modal end publication-request-modal components

erikap commented:
- Ok voor mij om `uploadedPieces` te hernoemen naar `pieces`. Dekt inderdaad beter de lading.

- `createPublicationRequestActivity` op de publication-service kan je hernoemen naar `createPublicationRequest`, maar de andere zou ik zo laten. Het idee met de prefix `on...`  is dat de component-hook altijd start met `on...` of `did...` (vb. `onClose` of `didSave`), maar de actie nooit (vb. `save` of `closeRegistrationModal`). Die zou ik dus zeker niet gelijktrekken.

Commit:
aside/rename @ErikaP `createXRequest`
KAS-3190 pub: v3: proofs: request publication from ProofReceivedPanel

Commit:
Merge branch 'feature/KAS-3190-pub-v3--proofs-pub-req' of github.com:kanselarij-vlaanderen/frontend-kaleidos into feature/KAS-3190-pub-v3--proofs-pub-req

Commit:
remove useless array copy
> pieces getter does exactly the same
KAS-3190 pub: v3: proofs: request publication from ProofReceivedPanel

Commit:
PR|aside/rename `pieces`
KAS-3190 pub: v3: proofs: request publication from ProofReceivedPanel

Commit:
PR | aside/rename `edit`
KAS-3190 pub: v3: proofs: request publication from ProofReceivedPanel

Commit:
Merge branch 'development' into feature/KAS-3190-pub-v3--proofs-pub-req

Commit:
Merge branch 'development' into feature/KAS-3190-pub-v3--proofs-pub-req

Commit:
PR | aside/comment createProofRequest
KAS-3190 pub: v3: proofs: request publication from ProofReceivedPanel

erikap reviewed (approved):
Ik heb voor de zekerheid de Jenkins run nog eens manueel getriggerd, maar ik denk niet dat de oorzaak van de falende test uit de vorige run in deze PR zit.

erikap commented:
Het zijn andere testen, niet gerelateerd aan deze PR, die falen nu. Ik ga deze mergen.
